coverage/src/main/java/gov/cms/ab2d/coverage/repository/CoverageServiceRepository.java

```diff
@@ -572,14 +598,17 @@ private List<CoverageMembership> queryCoverageMembership(CoveragePagingRequest p
 
         pageCursor.ifPresent((cursor) -> sqlParameterSource.addValue("cursor", cursor));
 
+        boolean isOptOutOn = propertiesService.isToggleOn("OptOutOn", false);
         // Grab the enrollment
         List<CoverageMembership> enrollment;
         NamedParameterJdbcTemplate template = new NamedParameterJdbcTemplate(dataSource);
         if (pageCursor.isPresent()) {
-            enrollment = template.query(SELECT_COVERAGE_WITH_CURSOR, sqlParameterSource,
+            String queryWithCursor = (isOptOutOn) ? SELECT_OPTOUT_COVERAGE_WITH_CURSOR : SELECT_COVERAGE_WITH_CURSOR;
```

Comment:
Whats your thought about moving this if , else logic to a different function to reduce code repetition?

Reply:
I don't see how to recode code repetition, because different function will have the same if condition with 2 booleans: pageCursor and OptOutOn..